studio/components/interfaces/Settings/Database/SSLConfiguration.tsx

```diff
@@ -77,7 +79,7 @@ const SSLConfiguration = () => {
         </div> */}
       </div>
       <FormPanel>
-        {hasAccessToSSLEnforcement && (
+        {sslEnforcement && hasAccessToSSLEnforcement && (
```

Comment:
`sslEnforcement` flag is currently set only to team members in config cat @joshenlim 

or maybe i'm reading that wrong?

Reply:
yes that's correct - this is the SSL enforcement flag specifically

previously we were hiding incorrectly the whole SSL config section based on this flag
but this flag targets the SSL enforcement UI only

Reply:
but this is `false` for everyone apart from team. that correct?

Reply:
yup that's correct

Reply:
cool!